packages/connections/src/components/connections.tsx

```diff
@@ -54,16 +54,29 @@ const formContainerStyles = css({
   gap: spacing[4],
 });
 
+function getElectronAppName(): string {
+  const defaultAppName = 'MongoDB Compass';
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const electron = require('electron');
+    return electron?.remote?.app?.getName() || defaultAppName;
+  } catch (e) {
+    return defaultAppName;
+  }
+}
+
 function Connections({
   onConnected,
   connectionStorage = new ConnectionStorage(),
+  appName = getElectronAppName(),
```

Comment:
In `compass-home` we have appName and can pass it as a prop to `Connections`: https://github.com/mongodb-js/compass/blob/main/packages/compass-home/src/components/home.tsx#L257 
Maybe we can remove this defaulting code?